lib/sqlite/Makefile.am

```diff
@@ -3,7 +3,7 @@
 include $(top_srcdir)/Makefile.am.common
 
 if ENABLE_PTHREAD_SUPPORT
-AM_CPPFLAGS += -DSQLITE_THREADSAFE=1
+AM_CPPFLAGS += -DSQLITE_THREADSAFE=1 -DSQLITE_DEFAULT_FILE_PERMISSIONS=0600
```

Comment:
The equivalent change for Windows lib/sqlite/NTMakefile

diff --git a/lib/sqlite/NTMakefile b/lib/sqlite/NTMakefile
index f9e90fb77..f6daa119d 100644
--- a/lib/sqlite/NTMakefile
+++ b/lib/sqlite/NTMakefile
@@ -37,6 +37,10 @@ INCFILES=    \
        $(INCDIR)\sqlite3.h     \
        $(INCDIR)\sqlite3ext.h

+cdefines = $(cdefines) -DSQLITE_THREADSAFE=1 \
+       -DSQLITE_DEFAULT_FILE_PERMISSIONS=0600 \
+       -DSQLITE_WITHOUT_ZONEMALLOC=1
+
 $(LIBSQLITE): $(OBJ)\sqlite3.obj
        $(LIBCON)

Reply:
Oops, I wrote:

```# pmode 0600 = _S_IREAD | _S_IWRITE

$(OBJ)\sqlite3.obj: sqlite3.c $(INCFILES)
        $(C2OBJ_P) -DSQLITE_THREADSAFE=1 -DSQLITE_DEFAULT_FILE_PERMISSIONS=0600
```

Reply:
Fixed.  Please check.